semantic_release/dist.py

```diff
@@ -9,6 +9,18 @@
 logger = logging.getLogger(__name__)
 
 
+def should_build():
```

Comment:
Please add tests of these in `test_dist.py`

Reply:
Done